events/messageReactionAdd.js

```diff
@@ -23,4 +23,27 @@ module.exports = (client, reaction, user) => {
 			}
 		}
 	}
+
+
+	// AnonSuggestion reaction listening
+	const {modBot} = client.config.channelIDs;
+	const {suggestions} = client.config.channelIDs;
+
+	if (reaction.message.channel == client.channels.get(modBot)) {
```

Comment:
triple equals adds type checking and would use it here, and in most cases 
`if (reaction.message.channel === client.channels.get(modBot)) {`

Reply:
but I like muh abstract equality comparator, I've fixed it now.